sql/api/src/main/scala/org/apache/spark/sql/internal/types/AbstractStringType.scala

```diff
@@ -37,7 +37,7 @@ case object StringTypeBinary extends AbstractStringType {
 }
 
 /**
- * Use StringTypeBinaryLcase for expressions supporting only binary and lowercase collation.
+ * Use StringTypeBinaryLcase for expressions supporting only binary and binary lowercase collation.
```

Comment:
Isn't `binary lowercase` here misleading in a sense that it implies we do trivial `lower` then binary compare?

Reply:
Ok agree.